x-pack/plugins/cases/server/routes/api/cases/delete_cases.ts

```diff
@@ -16,7 +16,10 @@ export const deleteCaseRoute = createCasesRoute({
   path: CASES_URL,
   params: {
     query: schema.object({
-      ids: schema.arrayOf(schema.string()),
+      ids: schema.arrayOf(schema.string({ minLength: 1 }), {
+        minSize: 1,
+        maxSize: MAX_DELETE_CASE_IDS,
```

Comment:
Limiting the deletion API based on the issue here: https://github.com/elastic/kibana/issues/146945